### PR TITLE
Croatian translation

### PR DESCRIPTION
**Pull request by sheeepdev**
Hi!

This is my first time doing any translation-work for GNOME apps and `.po` in general so I hope I did everything right.
Keep in mind some things could not be translated so that it would make sense, e.g "logs" cant really be translated so I incorporated the english word. I didn't know to use `hr_HR` (Croatian (Croatia)) or `hr` (just Croatian) but I ended up using `hr_HR`, it's kinda weird because people also use the language in Bosnia and Herzegovina and a few locales exist.

Also sorry for the double commits, I used poedit which created an additional `.mo` file which I see is not needed so I commited two times with the same title accidentally 😅 

Thanks

### CODE REVIEW

File: po/LINGUAS [hunk 1]
```diff
@@ -1 +1 @@
- ru
\ No newline at end of file
+ru hr_HR
```
Comment (knuxify) [inline po/LINGUAS:1]: iirc, the new language should be included on a separate line

File: po/hr_HR.po [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
+"n%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
+"X-Generator: Poedit 3.1\n"
+
+# Translators: Do NOT translate or transliterate this text
+#: data/app.drey.EarTag.desktop.in:3 data/app.drey.EarTag.appdata.xml.in:3
+msgid "Ear Tag"
+msgstr ""
```
Comment (knuxify) [inline po/hr_HR.po:25]: you can just copy the "Ear Tag" string into the msgstr here

**Review by knuxify — CHANGES_REQUESTED** [review]

**Comment by knuxify** [comment]
Other than that, LGTM! Thanks for the contribution!

**Comment by sheeepdev** [comment]
> Other than that, LGTM! Thanks for the contribution!

Fixed!

**Pull request merged**